packages/@aws-cdk/core/README.md

```diff
@@ -352,13 +343,10 @@ response to the CloudFormation service and handle various error cases.
 Set `serviceToken` to `lambda.functionArn` to use this provider:
 
 ```ts
-import * as lambda from '@aws-cdk/aws-lambda';
-import { CustomResource } from '@aws-cdk/core';
-
-const fn = new lambda.Function(this, 'MyProvider');
+const fn = new lambda.Function(this, 'MyProvider', functionProps);
 
 new CustomResource(this, 'MyResource', {
-  serviceToken: lambda.functionArn
+  serviceToken: fn.functionArn,
```

Comment:
Wait we're not linting these as well, are we?

Reply:
We are not, but I still apply the same conventions (this reduces "unrelated changes" in the future, should we need to add more properties)